src/gtest/main.cpp

```diff
@@ -30,9 +30,7 @@ int main(int argc, char **argv)
   libsnark::default_r1cs_ppzksnark_pp::init_public_params();
   libsnark::inhibit_profiling_info = true;
   libsnark::inhibit_profiling_counters = true;
-  fs::path pk_path = ZC_GetParamsDir() / "sprout-proving.key";
-  fs::path vk_path = ZC_GetParamsDir() / "sprout-verifying.key";
-  params = ZCJoinSplit::Prepared(vk_path.string(), pk_path.string());
+  params = ZCJoinSplit::Prepared();

```

Comment:
As I understand, JoinSplit transactions in Zcash were used in Sprout only. 
Pastel uses Sapling from block 20, and there are no JoinSplit transactions in mainnet in these first 20 blocks. 
So, why we then even keep this ZCJoinSplit?

Reply:
I left it because in other case the task will require large refactoring